TestForSignUp/ElementFiller.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Data;
+using System.Text;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    class ElementFiller
```

Comment:
What access modifier here?

Reply:
internal(by default)

Reply:
> internal(by default)

Good practice to explicitly set access modifiers

Reply:
fixed